src/Input/Input.js

```diff
@@ -114,6 +124,10 @@ export default class Input extends Component {
      */
     inputClassName: PropTypes.string,
     /**
+     * If true, a textarea element will be rendered.
+     */
+    multiLine: PropTypes.bool,
```

Comment:
Yea, I actually had it as `multiline` originally but I noticed that in v0 it was `multiLine` so I refactored it to keep the API consistent. "multi-line" is one word, not two, so I think `multiline` would be more correct here. 

I know v1 is already going to be introducing breaking changes, so maybe we just change it to `multiline` here?